components/dialog/harvester/MigrationDialog.vue

```diff
@@ -48,7 +48,7 @@ export default {
 
       return nodes.filter((n) => {
         // do not allow to migrate to self node
-        return n.id !== this.vmi?.status?.nodeName && n.isMigrateable;
+        return n.id !== this.vmi?.status?.nodeName && n.isMigratable;
```

Comment:
This is probably the most interesting change

Reply:
There's a matching change made in the relevant resource type's model